extensions/tachigo-demo-sidepanel/src/styles/fonts.css

```
@@ -0,0 +1,39 @@
+@font-face {
+  font-family: 'Press Start 2P';
+  src: url('../assets/fonts/PressStart2P-Latin.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Zpix';
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Zpix" (font-family-name-quotes)

(font-family-name-quotes)